docs/user/contacts.md

```diff
@@ -0,0 +1,53 @@
+# Contacts
```

Comment:
I think `Contributing` is better than `Contacts`, @vankichi @hlts2 what do you think?